Makefile

```diff
@@ -1754,6 +1756,11 @@ $(bins): bin/%: bin/%.d | bin/prereqs bin/.submodules-initialized
 	mv -f $@.d.tmp $@.d
 	@$(GO_INSTALL) -v $(if $($*-package),$($*-package),./pkg/cmd/$*)
 
+$(xbins): bin/%: bin/%.d | bin/prereqs bin/.submodules-initialized
```

Comment:
The `$(bins)` target above prints a line saying what it's building (`@echo go install -v $*`). How about doing that here too? Something like:

```
    @echo go build -v $(GOFLAGS) $(GOMODVENDORFLAGS) -tags '$(TAGS)' -ldflags '$(LINKFLAGS)' -o $@ $*
```